### DISCUSSION TITLE
it is possible to have written who uses the commands?

### DISCUSSION THREAD

**Discussion by thenewgessiko** [moonstar-x/discord-tts-bot#77] [Q&A]
Hi, but in the discors tts bot, it is possible to have written who uses the commands / say / amazon_say etc ... and what was written?

**Comment by moonstar-x** [comment]
You want this to show up on Discord? LIke for everyone to see?

Currently the bot will log to the console who says what with that settings and what channel, if that's of any help to you.

**Comment by thenewgessiko** [comment]
yes I want you to read what you say using /say or /amazon_say etc ...

Il Sab 18 Giu 2022, 22:36 moonstar-x ***@***.***> ha scritto:

**Comment by moonstar-x** [comment]
By "yes I want you to read", you mean that you want the bot to say verbally:

"User X said Y"

?

If yes, you can enable the `enable_who_said` option in the configuration to do that.

**Comment by thenewgessiko** [comment]
Yes the voice who said is active and only says it in the voice channel, but
I also want it to say it in writing if it is possible

Il Dom 19 Giu 2022, 20:23 moonstar-x ***@***.***> ha scritto:

**Comment by moonstar-x** [comment]
Ah, currently it is not possible. The bot responding is currently private so you can't see what others say. I guess I could add the option for the bot to keep that in the channel in the future.

**Comment by thenewgessiko** [comment]
Oh. ok bro, i hope you will put it in the future, for now i'm satisfied so
that it says it in the voice chat, keep me updated if in the future you
will do it that you can read in the text channel

Il Dom 19 Giu 2022, 20:48 moonstar-x ***@***.***> ha scritto:
